src/screens/Settings/components/Email2FAToggle.tsx

```diff
@@ -55,6 +59,18 @@ export function Email2FAToggle() {
         onConfirm={enableEmailAuthFactor}
         confirmButtonCta={_(msg`Enable`)}
       />
+      <VerifyEmailDialog
+        control={verifyEmailDialogControl}
+        changeEmailControl={changeEmailDialogControl}
+        onCloseAfterVerifying={enableDialogControl.open}
+        reasonText={_(
+          msg`You'll need to verify your email address before enabling email 2FA.`,
```

Comment:
```suggestion
          msg`You need to verify your email address before you can enable email 2FA.`,
```
I think this might be slightly clearer